core/src/main/scala/kafka/admin/BrokerApiVersionsCommand.scala

```diff
@@ -17,32 +17,27 @@
 
 package kafka.admin
 
-import java.io.PrintStream
-import java.io.IOException
-import java.util.Properties
-import java.util.concurrent.atomic.AtomicInteger
-import java.util.concurrent.{ConcurrentLinkedQueue, TimeUnit}
-
-import kafka.utils.{CommandDefaultOptions, CommandLineUtils}
 import kafka.utils.Implicits._
-import kafka.utils.Logging
-import org.apache.kafka.common.utils.Utils
-import org.apache.kafka.clients.{ApiVersions, ClientDnsLookup, ClientResponse, ClientUtils, CommonClientConfigs, Metadata, NetworkClient, NodeApiVersions}
+import kafka.utils.{CommandDefaultOptions, CommandLineUtils, Logging}
 import org.apache.kafka.clients.consumer.internals.{ConsumerNetworkClient, RequestFuture}
+import org.apache.kafka.clients._
+import org.apache.kafka.common.Node
 import org.apache.kafka.common.config.ConfigDef.ValidString._
 import org.apache.kafka.common.config.ConfigDef.{Importance, Type}
 import org.apache.kafka.common.config.{AbstractConfig, ConfigDef}
 import org.apache.kafka.common.errors.AuthenticationException
 import org.apache.kafka.common.internals.ClusterResourceListeners
+import org.apache.kafka.common.message.ApiVersionsResponseData.ApiVersionCollection
 import org.apache.kafka.common.metrics.Metrics
 import org.apache.kafka.common.network.Selector
-import org.apache.kafka.common.protocol.{ApiKeys, Errors}
-import org.apache.kafka.common.utils.LogContext
-import org.apache.kafka.common.utils.{KafkaThread, Time}
-import org.apache.kafka.common.Node
-import org.apache.kafka.common.message.ApiVersionsResponseData.ApiVersionCollection
-import org.apache.kafka.common.requests.{AbstractRequest, AbstractResponse, ApiVersionsRequest, ApiVersionsResponse, MetadataRequest, MetadataResponse}
+import org.apache.kafka.common.protocol.Errors
+import org.apache.kafka.common.requests._
+import org.apache.kafka.common.utils.{KafkaThread, LogContext, Time, Utils}
 
+import java.io.{IOException, PrintStream}
```

Comment:
> I'd like this order optimization. However, it is not related to this PR and it may produce conflicting files to other PRs. Hence, could you please revert this change?

@chia7712 Thank you so much for your comment. Now both IOException and PrintStream are inside the same {}. 
Please help me to review this PR again. :)

Reply:
Sorry for my unclear comment. My point was that this PR re-order all imports of this class. It would be better to revert all changes related to re-order.

Reply:
Sorry, blame me, I understand what you mean now, thank you for your feedback, I will make changes and submit it later today. :)